Notes-CaaSP.md

```diff
@@ -132,11 +132,177 @@ kubectl create secret generic ceph-secret-admin
+  openstack security group rule create cap --protocol tcp --dst-port 4443:4443 --remote-ip 0.0.0.0/0
+  openstack security group rule create cap --protocol tcp --dst-port 80:80 --remote-ip 0.0.0.0/0
+  openstack security group rule create cap --protocol tcp --dst-port 2222:2222 --remote-ip 0.0.0.0/0
+  ```
+
+* Edit `openstack.tfvars`. Use the names of the just created OpenStack objects
+
+  Example:
+  ```
+  image_name = "SUSE-CaaS-Platform-2.0-GM"
+  internal_net = "caasp-net"
+  external_net = "floating"
+  admin_size = "m1.large"
+  master_size = "m1.large"
+  masters = 1
+  worker_size = "m1.xlarge"
+  workers = 2
+  ```
+
+* Initialize terraform
+
+  ```
+  terraform init
+  ```
+
+
+### Deploy CaaSP
+
+* Source the openrc.sh file, set the project and deploy
+
+  ```
+  . openrc.sh
+  export OS_PROJECT_NAME='caasp'
+  ./caasp-openstack apply
+  ```
+
+* Wait for 5 - 10 minutes until all systems are up and running
+* Get an overview of your CaaSP installation
+
+  ```
+  openstack server list
+  ```
+
+* Add the initial created `cap` security group to all CAP workers
+
+  ```
+  openstack server add security group caasp-worker0 cap
+  openstack server add security group caasp-worker1 cap
+  ```
+
+* Access to CaaSP nodes
+
+  For CAP you might have to log into the CaaSP master and nodes. To do so,
+  use ssh with the ssh key in the `automation/caasp-openstack-terraform/ssh`
+  dir to login as root.
+
+
+### Bootstrap CaaSP
+
+* Point your browser at the IP of the CaaSP admin node
+* Create a new admin user
+* On `Initial CaaS Platform Configuration`
+  * _Admin node_ - the prefilled value (public/floating ip) needs to be replaced by the internal openstack caasp subnet ip of the CaaSP admin node
+  * Enable the "Install Tiller" checkbox
+* On `Bootstrap your CaaS Platform`
+  * Click [Next]
+* On `Select nodes and roles`
+  * Click [Accept All nodes] and wait until they appear in the upper part of the page
+  * Define master and nodes
+  * Click [Next]
+* On `Confirm bootstrap`
+  * _External Kubernetes API FQDN_ - Enter the public(floating) IP from the CaaSP master with added .xip.io domain suffix
+  * _External Dashboard FQDN_ - Enter the public(floating) IP from the CaaSP admin with added .xip.io omain suffix
+
+### Prepare CaaSP for CAP
+
+* Commands to run on the CaaSP master
+
+  This is only necessary if you use hostpath as storage class
+  ```
+  perl -p -i -e 's@^(KUBE_CONTROLLER_MANAGER_ARGS=)"(.*)"@\1"\2 --enable-hostpath-provisioner"@' /etc/kubernetes/controller-manager
+  mkdir -p /tmp/hostpath_pv
+  chmod a+rwx /tmp/hostpath_pv
+  systemctl restart kube-controller-manager.service
+  ```
+
+* Commands to run on the CaaSP nodes
+
+  This is only necessary if you use hostpath as storage class
+  ```
+  mkdir -p /tmp/hostpath_pv
+  chmod a+rwx /tmp/hostpath_pv
+  ```
+
+  Resize your root filesystem of the worker to match the disk provided by OpenStack (https://bugzilla.suse.com/show_bug.cgi?id=1069471#c12)
```

Comment:
I would remove the link to the Bugzilla entry because it's not accessible for everybody.